app/Dockerfile

```diff
@@ -35,7 +36,8 @@ RUN pip3 install --no-cache-dir pyinstaller \
     && pip3 install --no-cache-dir patchelf==0.17.0.0 \
     && pip3 install --no-cache-dir staticx \
     && pip3 install --no-cache-dir -r ./app/requirements.txt \
-    && pip3 install --no-cache-dir -r ./app/requirements-links.txt
+    && pip3 install --no-cache-dir -r ./app/requirements-links.txt \
+    && pip3 install --no-cache-dir /vehicle_model
```

Comment:
This I don't understand fully:
* This is for app runtime container, right?
* When/where is that step done for local dev / CI?

Reply:
this is explicitly for running the app in a container, yes!
For local execution the app/sourcecode has access to the ./gen/vehicle_model

If you start the k3d runtime and don't install the model upfront you get the following:

`kubectl logs sampleapp-8489757bf-mgd6z
Defaulted container "sampleapp" out of: sampleapp, daprd
sdv/dapr/_servicier.py:55: DeprecationWarning: There is no current event loop
Traceback (most recent call last):
  File "main.py", line 28, in <module>
ModuleNotFoundError: No module named 'vehicle'
[9] Failed to execute script 'main' due to unhandled exception!`

Reply:
Ok, installation as python package is done in the devenv-devcontainer-setup in /vehicle-model-lifecycle/src/generate_model.py